learning_paths/api/v1/urls.py

```diff
@@ -25,4 +28,11 @@
         LearningPathUserGradeView.as_view(),
         name="learning-path-grade",
     ),
+    path(
+        "learning-paths/<uuid:learning_path_id>/enrollments/",
+        LearningPathEnrollmentView.as_view(),
+        name="learning-path-enrollments",
+    ),
+    path("enrollments/", FetchEnrollmentsView.as_view(), name="fetch-enrollments"),
+    path("enrollments/bulk-enroll/", BulkEnrollAPIView.as_view(), name="bulk-enroll"),
```

Comment:
Tiny nit: we should update the name from `bulk_enroll` to `bulk-enroll` in `0003-learning-path-enrollment-api.rst`.